funcx_sdk/funcx/sdk/login_manager/manager.py

```diff
@@ -78,14 +78,29 @@ def run_login_flow(
 
         do_link_auth_flow(self._token_storage, scopes)
 
-    def logout(self) -> None:
+    def logout(self, resource_server: str = None) -> bool:
+        """
+        If resource_server is specified, this will only revoke the
+        specified server tokens.  Returns True if any tokens were found
+         and revoked.
+
+        TODO for Reviewers:  Should we add capability to log out of
+        specific resource servers?  ie. for FuncX Endpoints the resource
+        servers are auth.globus.org, funcx_service, and search.api.globus.org
+        If not, the new resource_server param will be removed until
+        future feature needs
+        """
```

Comment:
I would vote not adding the resource server complexity.  That ability is supported from the SDK directly if someone needs that fine grained control.  But, for the most part, what we want here is a way for someone to be confident the authentication they had acquired by running endpoint commands is no longer valid.

Reply:
Yes I agree and resource_server param will be removed in next commit.